.github/workflows/ci-nix.yml

```diff
@@ -113,6 +114,8 @@ jobs:
         build:
           - aarch64-android
           - x86_64-android
+          - aarch64-apple-ios
```

Comment:
You want to build *to* ios? I don't think we support it and it's a bit different change.

Reply:
OK dropped it